tests/test_h1_client.c

```diff
@@ -326,6 +326,59 @@ H1_CLIENT_TEST_CASE(h1_client_request_send_body_chunked) {
     return AWS_OP_SUCCESS;
 }
 
+H1_CLIENT_TEST_CASE(h1_client_request_send_chunked_trailer) {
+    (void)ctx;
+    struct tester tester;
+    ASSERT_SUCCESS(s_tester_init(&tester, allocator));
+    struct aws_http_headers *trailers = aws_http_headers_new(allocator);
+    const struct aws_http_header trailer = {
+        .name = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("chunked"),
+        .value = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("trailer"),
+    };
```

Comment:
trivial: add another header, since there's only 1 test may as well stress what you can

Reply:
Good point, since this was actually broken for a while and the test didn't catch it. I caught it when running through the debugger. This debugger thing is pretty useful.